Code/Framework/AzFramework/AzFramework/Visibility/VisibleGeometryBus.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/ComponentBus.h>
+#include <AzCore/Component/Entity.h>
+#include <AzCore/Math/Matrix4x4.h>
+#include <AzCore/Memory/SystemAllocator.h>
+#include <AzCore/std/containers/vector.h>
+
+namespace AZ
+{
+    class ReflectContext;
+}
+
+namespace AzFramework
+{
+    //! VisibleGeometry describes details about visible geometry surfaces stored as generic indexed triangle lists
```

Comment:
Should the format (triangle list) be specified somewhere, so that you could eventually support lines, splines and points too?

Reply:
My current use case explicitly required triangle lists but there's no reason we can't support different formats for other use cases.